improver/nbhood/nbhood.py

```diff
@@ -428,10 +428,7 @@ def __init__(
                 DEFAULT_PERCENTILES.
         """
         super().__init__(radii, lead_times=lead_times)
-        try:
```

Comment:
I tried to find a case where the exception was raised so I could test it. It's not raised for a list of percentiles, a single percentile or the default percentiles. Maybe this was needed for an earlier version of python? Anyway as I couldn't test it I decided to remove it.